### ISSUE TITLE
Fatal memory error while reimporting vrm

### ISSUE THREAD

**Issue by vovan-ivanoff** [vrm-c/UniVRM#479]
![bug](https://user-images.githubusercontent.com/63170128/87872640-14068a00-c9c3-11ea-876d-5b5e9bb5d2e2.png)
I imported fbx into unity, then exported it in vrm with UniVRM an wanted to import this vrm again(as described in UniVRM wiki) and unity crashes with fatal memory error.

**To Reproduce**

1. open unity project, which i attached to this issue (google Drive)
2. click on VRM->UniVrm->import
3. import "avatar5.vrm"
4. and it all crashes

there is google drive link: https://drive.google.com/file/d/14uVRPKjbkBJYztDULn0Ok7WTa5g-gtia/view?usp=sharing

**Expected behavior**

i expected to get something like on this screenshot
![expected](https://user-images.githubusercontent.com/63170128/87872839-b5daa680-c9c4-11ea-85fc-29dd79f11e0d.png)

**Screenshots**

all screenshots in youtube video https://youtu.be/56kk5d75qo0

**Environments:**

 - OS: Windows 10(16GB of RAM)
 - Unity version: Unity-2018.4.21f1
 - UniVRM version: 0.56.3

**Additional context**

If you want to see more description see video, that i uploaded

**Comment by PoChang007** [comment]
Thanks for your report.

I just tested your model and got the same error. I think the model's size is too big (blendshape takes too much space). So I tried to separate blendshape mesh (.fbx) by using [MeshUtility](https://github.com/vrm-c/UniVRM/tree/master/Assets/MeshUtility) before export. It works fine on vrm import.

Another trick is loading your .vrm model in runtime (SimpleViewer in UniVRM-Sample package ) and make it T-pose. Then click on `VRM` -> `UniVRM-0.56.3` -> `Export` as humanoid and select `Reduce BlendShape` and `Reduce BlendShape Clip` , or use MeshUtility before export.  when .vrm is imported again, this issue should be resolved.

Here is the size reduced version:
[аватар3_issue479.zip](https://github.com/vrm-c/UniVRM/files/4945586/3_issue479.zip)
